src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int sumSupply = 0;
+        int sumBuy = 0;
+        File fileRead = new File(fromFileName);
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileRead));
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(",");
+                value = reader.readLine();
+            }
+            String resultBuilder = builder.toString();
+            String[] data = resultBuilder.split(",");
```

Comment:
You can improve your code by using constants for strings (such as ","), and for magic numbers.